### ISSUE TITLE
Unable to install Ansible on Ubuntu 16 using DEB package - requires python-support

### ISSUE THREAD

**Issue by jladdjr** [ansible/ansible#18076]
The Tower team uses apt to install the devel version of Ansible on test instances. Installing Ansible on Ubuntu 16 results in a dependency error, though: 

```
The following packages have unmet dependencies:
 ansible : Depends: python-support (>= 0.90.0) but it is not installable
```
##### ISSUE TYPE
- Bug Report
##### COMPONENT NAME

Installer
##### ANSIBLE VERSION

Able to reproduce this on devel and stable-2.1. Have not tested on stable-1.9.

```
ansible 2.3.0 (devel 5ece97ae5b) last updated 2016/10/17 16:24:09 (GMT -400)
  lib/ansible/modules/core: (detached HEAD 500736d301) last updated 2016/10/17 16:24:09 (GMT -400)
  lib/ansible/modules/extras: (detached HEAD 9897c502a6) last updated 2016/10/17 16:24:09 (GMT -400)
  config file = /Users/jladd/.ansible.cfg
  configured module search path = Default w/o overrides

ansible 2.1.2.0 (stable-2.1 28016bf1a0) last updated 2016/10/17 16:52:43 (GMT -400)
  lib/ansible/modules/core: (detached HEAD ca822ec38c) last updated 2016/10/17 16:52:44 (GMT -400)
  lib/ansible/modules/extras: (detached HEAD 342a9e53ca) last updated 2016/10/17 15:18:53 (GMT -400)
  config file = /Users/jladd/.ansible.cfg
  configured module search path = Default w/o overrides
~/ansible/devel ~/ansible/devel/packaging/debian ~/git/ansible
```
##### CONFIGURATION

No special configuration
##### OS / ENVIRONMENT

Attempting to install on Ubuntu 16 (16.04.1 LTS (Xenial Xerus))
##### STEPS TO REPRODUCE
1. On Ubuntu 16 instance in ec2 (ami-e3c3b8f4), add repo that points to internal DEB build for Ansible.
2. Run `sudo apt-get install ansible`
##### EXPECTED RESULTS

Successful installation.
##### ACTUAL RESULTS

Following error:

```
root@ip-10-233-104-228:/etc# apt-get install ansible
Reading package lists... Done
Building dependency tree
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 ansible : Depends: python-support (>= 0.90.0) but it is not installable
E: Unable to correct problems, you have held broken packages.
```

**Comment by jctanner** [comment]
@jladdjr just to clarify, the deb package was built with the ansible makefile's deb target?

needs_info

**Comment by abadger** [comment]
https://wiki.ubuntu.com/MeetingLogs/devweek1107/PythonPackaging so perhaps python-support has been removed and packages for Ubuntu16+ are going to need dh_python2 instead?

**Comment by jladdjr** [comment]
@jctanner - the deb was built using this job:
http://jenkins.testing.ansible.com/view/Ansible/job/Build_Ansible_DEB/configure

Looks like for an unofficial build (which applies here), it calls `deb-upload`

**Comment by jctanner** [comment]
I did the following without error:
1. spun up a geerlingguy/ubuntu1604 vagrant box
2. installed all the various make requirements: asciidoc python-setuptools build-essential devscripts cdbs deb-helper
3. ran "make deb"
4. copied the resulting deb to a local apt repo and added to sources per: http://askubuntu.com/a/176546
5. apt-get update
6. apt-get install ansible

```
root@vagrant:/usr/local/mydebs# apt-get install ansible
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following NEW packages will be installed:
  ansible
0 upgraded, 1 newly installed, 0 to remove and 90 not upgraded.
Need to get 0 B/1,615 kB of archives.
After this operation, 12.7 MB of additional disk space will be used.
WARNING: The following packages cannot be authenticated!
  ansible
Install these packages without verification? [y/N] y
Get:1 file:/usr/local/mydebs ./ ansible 2.3.0-100.git201610181727.01b75f9.devel~unstable [1,615 kB]
Selecting previously unselected package ansible.
(Reading database ... 109427 files and directories currently installed.)
Preparing to unpack .../ansible_2.3.0-100.git201610181727.01b75f9.devel~unstable_all.deb ...
Unpacking ansible (2.3.0-100.git201610181727.01b75f9.devel~unstable) ...
Processing triggers for man-db (2.7.5-1) ...
Setting up ansible (2.3.0-100.git201610181727.01b75f9.devel~unstable) ...
Installing new version of config file /etc/ansible/ansible.cfg ...

root@vagrant:/usr/local/mydebs# ansible --version
ansible 2.3.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides
```

**Comment by jctanner** [comment]
per slack conversation, this issue most likely stems from building the package on 14.04 and then trying to install on 16.04. The python build environment is going to differ enough that it isn't going to work. Typical problem with packaging in an environment that differs from the target.

**Comment by jladdjr** [comment]
Roger that - we'll take some time to look into how we're doing the packing

**Comment by jctanner** [comment]
@jladdjr can we close this?

needs_info

**Comment by jladdjr** [comment]
@shanemcd has looked into how the Ansible DEB is being built and saw why our jenkins slave was trying to use the python support package. 

It looks like a `dpkg-query` ([here](https://github.com/ansible/ansible/blob/4549604cc73321c079ad0eec53dd3f898c63be4b/packaging/debian/rules#L5)) was looking for python-support on the system. Shane confirmed that the query reported that the package was present even when the package had been removed and was only left in the cache. 

@shanemcd is looking into changing the build process to use pbuilder to build the package. Talked with @jctanner and Shane, we decided to close this ticket out and have Shane submit a PR if needed.
